packages/cli/src/utils/verifyManifest.ts

```diff
@@ -76,6 +76,28 @@ const endpointCallSchema = object({
 const hostnameRegex =
     /^(([\dA-Za-z]|[\dA-Za-z][\dA-Za-z-]*[\dA-Za-z])\.)*([\dA-Za-z]|[\dA-Za-z][\dA-Za-z-]*[\dA-Za-z])$/;
 
+const ScopeEnum = z.enum([
+    'basic:read',
+    'basic:write',
+    'account:read',
+    'webhook:read',
+    'webhook:write',
+]);
+
+const permissionsSchema = z.object({
+    permissions: z
+        .object({
+            scopes: z
+                .array(ScopeEnum)
+                .min(1, 'At least one scope is required')
+                .max(3, 'No more than 3 scopes are allowed')
```

Comment:
Only 3 are allowed? Does that includes 'basic:read'?

Reply:
ups thx, i didnt include the workflow first xD